projects/plugins/social/src/class-jetpack-social.php

```diff
@@ -153,7 +153,7 @@ public function initial_state() {
 		$shares                  = $publicize->get_publicize_shares_info( Jetpack_Options::get_option( 'id' ) );
 		$refresh_plan_from_wpcom = true;
 		$show_nudge              = ! Current_Plan::supports( 'social-shares-1000', $refresh_plan_from_wpcom );
-
+		$parsed_url              = wp_parse_url( get_site_url() );
```

Comment:
Do you think you could use the existing [`get_site_suffix`](https://github.com/Automattic/jetpack/blob/trunk/projects/packages/status/src/class-status.php#L328) function for this? I think it gives you the same result but with some extra checks to make sure it's correct.

Something like `'siteSuffix' => ( new Status() )->get_site_suffix()`